site/src/pages/WorkspacePage/WorkspacePage.test.tsx

```diff
@@ -284,65 +280,4 @@ describe("WorkspacePage", () => {
       })
     })
   })
-
-  describe("Resources", () => {
```

Comment:
I really think tests like these should live in Storybook. Integration tests should be used for user interactions IMO.